### PR TITLE
Add squad by code

### PR DESCRIPTION
**Pull request by abhishyantkhare**
Add the ability to add yourself to a squad by code. This has the following steps:

Added a squad code column on the db as well as generating the squad code using a uuid. Each time a squad is created we generate a new uuid for it.

We display the uuid for the squad:
image

Users can then add the squad on the add squad page (we show the done button when someone has typed in something):
image

### CODE REVIEW

File: mobile/events/events.tsx [hunk 1]
```diff
@@ -15,10 +15,11 @@ export type Event = {
   url?: string
 }
 
+const SQUAD_CODE_TITLE_TEXT = "Squad Code: "
 
 const Events = (props) => {
   const [events, setEvents] = useState([])
-  const [squadId, setSquadId] = useState(props.route.params.squadId)
```
Comment (vivekjain1397) [inline mobile/events/events.tsx:21]: We still need the squadId to retrieve events so I don't think you should remove this. Squad code should be different.

File: backend/migrations/versions/29a95de4aea6_.py [hunk 2]
```diff
@@ -0,0 +1,30 @@
+"""empty message
```
Comment (c0dyzeng) [inline backend/migrations/versions/29a95de4aea6_.py:1]: Let's add a message for the migration.

File: mobile/events/events.tsx [hunk 3]
```diff
@@ -68,12 +69,30 @@ const Events = (props) => {
   }
 
 
-  const renderEventItem = ({ item }: { item: Event }) => {    
+
+  const renderSquadCode = () => {
+    console.log("SQUAD CODE")
+    console.log(squadCode)
```
Comment (c0dyzeng) [inline mobile/events/events.tsx:75]: We can probably remove these logs

**Comment by c0dyzeng** [comment]
These screens look fire 🔥

File: mobile/events/events.tsx (hunk 1 above)
Comment (abhishyantkhare) [reply mobile/events/events.tsx:22]: removed

File: backend/migrations/versions/29a95de4aea6_.py (hunk 2 above)
Comment (abhishyantkhare) [reply backend/migrations/versions/29a95de4aea6_.py]: donezo

File: mobile/events/events.tsx (hunk 3 above)
Comment (abhishyantkhare) [reply mobile/events/events.tsx]: donezo

**Pull request merged**